README.md

```diff
@@ -20,31 +20,11 @@ Once your environment is up and running, start the dev server:
 $ yarn start
 ```
 
-This will launch a server at http://127.0.0.1:8000/ but this can be
-configured in your `webpack.local.config.js` file.  Simply add
-something like this to that file if it doesn't exist:
+By default, this will launch a development server at http://127.0.0.1:8000/ but
+this can be configured (see Configuration below)
 
-```js
-module.exports = {
-    host: '0.0.0.0',
-    port: 8112,
-    baseUrl: 'http://localhost',
-    firebaseConfig: {...}
-};
-```
-
-The `baseUrl` needs to be set to be able to make calls to the backend.  As an
-alternative to using a local config file, you can set the `WEBSHOP_BASE_URL`
-environment variable.
-
-The `firebaseConfig` needs to be set to be able to connect to Firebase.  As an
-alternative to using a local config file, you can also set the
-`WEBSHOP_FIREBASE_CONFIG` environment variable.  The config needs to be valid
-JSON, so remember when copying the config from the Firebase console to change
-the keys to be double-quoted strings (this is only necessary when using an
-environment variable).
-
-When making changes, always format Elm files using the following `elm-format` release:
+When making changes, always format Elm files using the following `elm-format`
+release:
```

Comment:
Utvide dette til å si du kan installere det fra @atb-as/elm-format?

Reply:
`yarn global add @atb-as/elm-format` holder?

Reply:
Jepp. Det er også lagt til som lokal avhengighet for elm-review, men i tilfeller hvor man har verktøy som er avhengig av global installasjon.